src/components/SkyBrowser/SkybrowserTabs.jsx

```diff
@@ -0,0 +1,422 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '../common/Input/Button/Button';
+import MaterialIcon from '../common/MaterialIcon/MaterialIcon';
+import ScrollOverlay from '../common/ScrollOverlay/ScrollOverlay';
+import CenteredLabel from '../common/CenteredLabel/CenteredLabel';
+import Row from '../common/Row/Row'
+import NumericInput from '../common/Input/NumericInput/NumericInput'
+import ColorPickerPopup from '../common/ColorPicker/ColorPickerPopup'
+import Checkbox from '../common/Input/Checkbox/Checkbox'
+
+import { excludeKeys } from '../../utils/helpers';
+import TooltipSkybrowser from './TooltipSkybrowser';
+import SkybrowserFocusEntry from './SkybrowserFocusEntry';
+import styles from './SkybrowserTabs.scss';
+import { Resizable } from 're-resizable';
+
+class SkybrowserTabs extends Component {
+    constructor(props) {
+        super(props);
```

Comment:
This file uses 4 indentations everywhere whereas most of the other just use 2 spaces. Should be fixed imo